index.js

```diff
@@ -53,7 +55,13 @@ class Rapptor {
   }
 
   async stop() {
-    await this.server.stop({ timeout: 5 * 1000 });
+    const tags = ['server', 'stopping', 'notice'];
```

Comment:
Just append

Reply:
Done, Github didn't update this review note because the actual change is on the lines below.